src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,443 @@
+
+Figure 1: Overview of a common PET experiment, including blood measurements,
+and defined on a common time scale.
+Note, “time zero” can either be defined as time of injection or scan start,
+and all the PET and blood data should be decay-corrected to this time point.
+Furthermore, although in this example tracer injection coincides with scan start,
+this is not always the case and hence we allow for the flexibility of specifying
+either time of injection or scan start as “time zero”.
+
+PET data belong to the `pet` directory. PET imaging data SHOULD be stored in 4D
+(or 3D if only one volume was acquired) NIfTI files with `_pet` suffix.
+Volumes should be stored in chronological order (the order they were acquired in).
+
+Following the [BIDS Common Principles](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html), here we summarize how different labels are organized for PET data. Please, check the Common Principles section for further details.
+**Sessions**: Multiple sessions (visits) are encoded by adding an extra layer of directories
+and file names in the form of `ses-<label>`.
+Session labels can consist only of alphanumeric characters [a-zA-Z0-9]
+and should be consistent across subjects.
+If numbers are used in session labels we recommend using zero padding
+(for example, `ses-01`, `ses-11` instead of `ses-1`, `ses-11`).
+This makes results of alphabetical sorting more intuitive.
+The extra session layer (at least one `ses-<label>` subfolder) should be added
+for all subjects if at least one subject in the dataset has more than one session.
+Skipping the session layer for only some subjects in the dataset is not allowed.
+If a `ses-<label>` subfolder is included as part of the directory hierarchy,
+then the same `ses-<label>` tag must also be included as part of the file names themselves.
+In general, we assume that a new session with respect to PET starts with either a new injection
+(probably most common case) or with the subject being taken out of the scanner
+(same injection, but subject leaves the scanner and returns).
+However, for example, if a subject has to leave the scanner room and
+then be re-positioned on the scanner bed, for example, to use the bathroom,
+the set of PET acquisitions will still be considered as a session and match
+sessions acquired in other subjects.
+Similarly, in situations where different data types are obtained over several
+visits (for example, FDG PET on one day followed by amyloid PET a couple days after)
+those can be grouped in one session.
+Please see the difference with the `run-<label>` below.
+
+**Task:** With respect to the `task-<label>`, data is arranged in a similar way as task-based
+and resting state BOLD fMRI data.
+In case of studies using combined PET/fMRI, subject-specific tasks may be carried out
+during the acquisition within the same session.
+Therefore, it is possible to specify `task-<label>` in accordance with the fMRI data.
```

Comment:
it is not totally clear to me from this whether `task-<label>` is always required, with `resting` or `rest` in case there is no explicit task, or whether it can be left out (as with an anatomical MRI).

Reply:
It should be left out if there is no explicit task. Where in the MRI specification is this stated?

Reply:
So the setting we are thinking of is if you have an acquisition in the PET MRI, there might be a task measurement you want to associated the PET data with. But if you don't have a PET/MR or you use the machine like you would use a separate MRI and PET, then this is not necessary, therefore we wrote "it is possible". But the label is not always required. So this should be as with anatomical MRI. Should we then rephrase the wording or refer to anatomical MRI?

Reply:
in case you do a functional PET scan, e.g. with H2O15, but not using a PET/MR system, you would also want `task-<label>`, right? I don't see how it depends on hardware peculiarities (the PET camera being combined with an MR system), to me it seems that its presence is required as soon as the PET data is to be interpreted similar to an MRI `func`, but not if the data is to be interpreted as `anat`.